app/build.gradle

```diff
@@ -39,8 +39,6 @@ dependencies {
     implementation("io.micronaut.data:micronaut-data-hibernate-jpa")
     implementation("io.micronaut.sql:micronaut-jdbc-tomcat")
     implementation("io.micronaut.objectstorage:micronaut-object-storage-gcp")
-    implementation("io.micronaut.security:micronaut-security-jwt")
-    implementation("io.micronaut.security:micronaut-security-oauth2")
```

Comment:
I made comments below asking about the security annotations.  Does removing these two dependencies essentially remove `io.micronaut.security`?

Reply:
Correct, the dependency was removed since the auth service is going to be responsible for validating the JWT token. However, we can introduce if we decide to implement SecurityRule as mentioned in this comment https://github.com/UnityFoundation-io/Libre311/pull/42#discussion_r1483367272